### PR TITLE
Fixes #619: Fixed version not showing in Badgeyay-api

### LINKED ISSUE #619: Version not showing up in http://badgeyay-api.herokuapp.com/
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

`Version` is not showing up in the backend. Might be due to missing files.
**Expected behavior:**

The `version` showing on the website.

**Any newcomer can take up this**

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
Fixes #619 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:
- Added missing `js` files

### CODE REVIEW

**Comment by vaibhavsingh97** [comment]
Please review CC @ParthS007 @gabru-md @yashLadha

**Comment by vaibhavsingh97** [comment]
@gabru-md fixed

File: backend/app/static/js/script.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+                var deployLink = $('.version').attr('href', versionLink).html(version);
+            } else {
+                $('.version').html('Failed to access version');
+            }
+        },
+        error(error) {
+            $('.version').html('Failed to access version');
+        }
+    });
+});
```
Comment (ParthS007) [inline backend/app/static/js/script.js]: Add a line at EOF :+1:

Comment (vaibhavsingh97) [reply backend/app/static/js/script.js]: Done 👍

**Pull request merged**